synapsemonitor/monitor.py

```diff
@@ -100,7 +102,14 @@ def _find_modified_entities_file(syn: Synapse, syn_id: str, days: int = 1) -> li
     Returns:
         List of synapse ids
     """
-    raise NotImplementedError("Files not supported yet")
+    entity = syn.get(syn_id, downloadFile=False)
+    utc_mod = datetime.strptime(entity["modifiedOn"], "%Y-%m-%dT%H:%M:%S.%fZ")
+    local_mod = utc_mod.replace(tzinfo=tz.tzutc()).astimezone(tz.tzlocal())
+    local_now = datetime.now().replace(tzinfo=tz.tzlocal())
```

Comment:
If i'm reading this correctly, are you modifying the entity modified on date to local time then getting the current local time?  Can we just get the current UTC time and subtract?

Reply:
yes!  nice catch.  made the fix